src/app/modules/users/add-multiple-users/add-multiple-users.component.scss

```diff
@@ -0,0 +1,102 @@
+@import "../../../../assets/scss/variables.scss";
+.roundimg{
+    font-size: 10em;
+}
+.title {
+    font-weight: 600;
+    color: $white-color;
+    text-align: center;
+    margin-right: 12%;
+    padding: 5px 0px 0px 12px;
+}
+
+.titlehead {
+    font-weight: 600;
+    text-align: center;
+    margin-top: 3%;
+    margin-right: 18%;
+}
+.ftitle{
+    font-weight: 600;
+    color: $primary-color
+}
+.imagegap {
+    margin-top: 9%;
+    margin-right: 15%;
+}
+
+.task-header {
+    background-color: $primary-color;
+    // padding: 0 5px;
+    //  height: 50px;
+}
+.fx-spacer {
+    flex: 1 1 auto;
+}
+.space {
+    padding-top: 5%;
+}
+.actions {
+    padding-left: 50%;
+    padding-top: 40%;
+}
+
+::ng-deep .mat-dialog-container {
```

Comment:
Don't override the library class

Reply:
with the custom css it is not affecting the change